posit-dev/positron
# Merge 1.87.0 from upstream

### Intent

Merge 1.87.0 from upstream.

### Approach

Merge the upstream tag and resolve conflicts.

### QA Notes

See https://code.visualstudio.com/updates/v1_87 for a list of upstream changes. 

The code that computes and updates the console width had to be changed significantly to accommodate upstream changes; this feature should be double checked. 

Positron Help and Positron Data Explorer services now start a little later; we should ensure those still come up early enough to handle the initial state on start/reload. 

> [!NOTE]
>
> Converting to draft due to CI failures on Linux; investigating.
